app/views/student_review/_responses.html.erb

```diff
@@ -65,7 +65,8 @@
                               <%= link_to "Edit", {:controller => 'response', :action => 'edit', :id => @latest_response.id} %>
                           <%end%>
                         </td>
-                      <% else %>
+                      <!-- Update link is disabled for all calibrated reviews -->    
```

Comment:
This part looks weird to me. Why are update links disabled for calibrated reviews?

Reply:
Update link is disabled for calibrated review because calibration review happens only in one round unlike the peer reviews which take place in two or more rounds. So after submitting the review for calibration, the student will not be able to update the review and since there is no further round of calibration, we disabled the update link.